### PR TITLE
Rosemary/footer

### PR DESCRIPTION
**Pull request by liaorosemary**
Completed footer and added mail icon to home page.

Here's the original **_design_**:
image

Here are images of the **_actual_** homepage with footer in different sizing:

image
image
image
image

### CODE REVIEW

File: views/partials/footer.pug [hunk 1]
```diff
@@ -1,26 +1,16 @@
 div(id="footer")
     div(class="section")
-        b PHONE NUMBER 
-        br
-        p #{footer.phoneNumber}
+        b ©2023 Mathsoc
```
Comment (some-molasses) [inline views/partials/footer.pug:3]: do you think we could use `new Date().getFullYear()` to make this auto-update on an ongoing basis?

**Review by some-molasses — APPROVED** [review]
one small fix but otherwise approved!

File: views/partials/footer.pug (hunk 1 above)
Comment (liaorosemary) [reply views/partials/footer.pug]: Fixed. Please check the PR again :)

**Review by some-molasses — APPROVED** [review]
thanks!

**Pull request merged**